container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -1244,6 +1246,26 @@ func (manager *DockerManager) FetchLatestImage(ctx context.Context, dockerImage
 	return nil
 }
 
+func (manager *DockerManager) FetchImage(ctx context.Context, image string, download_mode image_download_mode.ImageDownloadMode) (bool, error) {
+	var err error
+	var pulledFromRemote bool = true
+
+	switch image_pulling := download_mode; image_pulling {
+	case image_download_mode.Always:
+		err = manager.FetchLatestImage(ctx, image)
+	case image_download_mode.Missing:
+		pulledFromRemote, err = manager.FetchImageMissing(ctx, image)
+	case image_download_mode.Never:
```

Comment:
I'm assuming this is where the `FetchLocalOnly` implementation would go?

Reply:
Yes, but we just decided to remove the `never` mode for now